ui/table/table.go

```diff
@@ -87,7 +87,13 @@ func (t Table) Print(w io.Writer) error {
 	writer := NewWriter(w, "-", t.BackgroundStr, t.BorderStr)
 	rowCount := len(t.Rows)
 	for _, section := range t.Sections {
+		// we're removing process rows from the instance count
```

Comment:
This comment seems a bit too specific to instances and processes while the `Table` struct is a more generic structure. I don't think the comment is really necessary since the `if` statement inside the `for` below is fairly clear.